src/core/network.js

```diff
@@ -75,6 +75,24 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if (FIREFOX || MOZCENTRAL)
+//var supportsMozChunked = true;
+//#endif
+//#if GENERIC
```

Comment:
I wanted to exclude the feature detection logic from the `CHROMIUM` builds. And AFAIK there is no `#elseif` in our preprocessor, hence I've used `#endif` + `#if GENERIC`.